cytominer_eval/evaluate.py

```diff
@@ -16,6 +16,9 @@
     mp_value,
 )
 
+# no idea why I need to do this, but otherwise it would give the module object error
+from cytominer_eval.operations.enrichment import enrichment
```

Comment:
Please add to [`cytominer_eval/operations/__init__.py`](https://github.com/cytomining/cytominer-eval/blob/master/cytominer_eval/operations/__init__.py):

```python
from .enrichment import enrichment
```

Then it should work just fine

Reply:
Done